### ISSUE TITLE
"Hide Search Matches" no longer worked

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#778]
Heya,

I was going to "add" this feature, but instead I found out, it's already there, but broken.

In doctools.js it says:

```
  $('<li class="highlight-link"><a href="javascript:Documentation.' +
    'hideSearchWords()">' + _('Hide Search Matches') + '</a></li>')
      .appendTo($('.sidebar .this-page-menu'));
```

but this is wrong: .sidebar should really be .sphinxsidebar

with this change, the link appears again and works as expected.

thank you for this excellent tool.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/778
- Originally reported by: Timo Paulssen
- Originally created at: 2011-10-02T12:06:55.175

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-10-02 15:51:12+00:00_

Fix #778: Fix "hide search matches" link on pages linked by search.

→ <<cset 975f9c7eaa7ec7a53a04a69f29b4cec9e8a4c78e>>

**Comment by shimizukawa** [comment]
_From Robert Lehmann on 2011-10-02 17:13:37+00:00_

Closes #778: Fix CSS class selector.

→ <<cset 478ffe5632ebd6a1b2b3a6fcdae214859288b5dd>>
